docs-gh-pages/consensus.md

```diff
@@ -0,0 +1,91 @@
+## Consensus
+
+### Architecture
+Every new `CheckpointBlock` is being created under a consensus round.
+
+### Glossary
+- **consensus owner** - the node which sent consensus proposal to others and initiated a new round
+- **facilitator** - node which has been selected to take part in a new round and responded successfully to participate in it (including owner)
+- **SOE** - signed observation edge
+- **data proposal** - transactions (and other checkpoint block data) proposed by the facilitator to include in the new consensus round
+
+### Consensus round
+1. Start consensus
+2. Broadcast and gather consensus data proposals
+3. Merge, broadcast and gather consensus block proposals
+4. Select majority block proposal and merge facilitators' signatures
+5. Accepted finished block and broadcast to non-facilitators
+
+### Stages
+
+#### 1. Start consensus
+`ConsensusScheduler` runs `consensusManager.startOwnConsensus()` every **n** seconds
+(*default value: 10s*, configurable under `constellation.consensus.start-own-interval`).
+To start a new round, node must be in one of valid node states. Otherwise, creating new round will be skipped.
+Valid node states for starting new consensus round are **Ready** and **SnapshotCreation**.
```

Comment:
If you use parts of code then I would make it `Ready` and `SnapshotCreation` instead of **Ready**  and **SnapshotCreation**